index.js

```diff
@@ -1,109 +0,0 @@
-const readline = require("readline");
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  let whichGame = await ask( //stores which game the user wants to play
-    'Do you (human) want to pick a number to guess, or should I (computer) pick a number for you (human) to guess?\nIf you want to pick a number, enter "1". If you want the computer to pick a number, enter "2".\n'
-  );
-  if (whichGame == 1) { //if else to determine which game is executed
-    let guess = 0; //Section to create variables that will be reassigned in the function.
-    let yOrN = 0;
-    let hOrL = 0;
-    let max = 100;
-    let min = 1;
-    let howManyTries = 0;
-    let whichGame = 0;
-
-    start();
-    async function start() {
-      console.log(
-        "Let's play a game where you (human) pick a number and I (computer) try to guess it.\n"
-      );
-      let max = await ask(
-        "What do you want the maximum number to be? I will guess numbers between this number and 1.\n"
-      );
-      console.log("You entered: " + max);
-      let secretNumber = await ask(
-        "What is your secret number?\nI won't peek, I promise...\n"
-      );
-      console.log("You entered: " + secretNumber);
-      console.log("I will now begin to guess your number. \n");
-      while (yOrN !== "Y") {
```

Comment:
an easy way to make the user input a little more module is to have `while(yOrN.toUpperCase() !== "Y")` This way if the user puts in a "y" it will still hit this section.